frontend/src/components/ide/AIAssistant/components/PendingChanges.jsx

```
@@ -0,0 +1,234 @@
+                        <span className="text-red-500">-{change.linesRemoved}</span>
+                    )}
+                </div>
+
+                {/* Status Icon */}
+                {getStatusIcon()}
+
+                {/* Actions */}
+                {change.status === 'pending' && (
+                    <div className="flex items-center gap-1" onClick={e => e.stopPropagation()}>
+                        <Button
+                            variant="ghost"
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)